Bundle/WidgetBundle/Builder/WidgetFormBuilder.php

```diff
@@ -136,7 +136,7 @@ public function renderNewQuantumForms($slot, View $view, $widgets, $activeWidget
     {
         $forms = [];
         foreach ($widgets as $key => $widget) {
-            $_quantum = null !== $quantum ? $quantum : $key;
+            $_quantum = null !== $quantum ? $quantum : $this->convertToString($key);
```

Comment:
$key could be equals to 'static' according to WidgetMapager.php:106, do this may cause a potential bug ?